.travis.yml

```diff
@@ -12,15 +12,9 @@ language: python
 
 matrix:
   include:
-    - python: '2.7'
-      env: TOXENV="flake8,flake8-docs"
-
     - python: '3.6'
       env: TOXENV="flake8,flake8-docs"
 
-    - python: '2.7'
```

Comment:
requires similar changes in https://github.com/MacPython/gensim-wheels/blob/master/.travis.yml and https://github.com/MacPython/gensim-wheels/blob/master/appveyor.yml (btw, don't forget to update `multibuild` first too)